### ISSUE TITLE
As a developer, I'd like to access a Data tool homepage to learn more about the API.

### ISSUE THREAD

**Issue by diego-** [18F/college-choice#258]
Tasks with links to specific stories:

- [x] #263: Framework / hosting env. for data tool homepage set up.
- [x] #161: Header / menu
- [x] #143: Text intro blurb at top of page (welcome, mission statement, etc.)
- [x] #161: API Documentation block
- [x] #157: Download the Data section with just the Download All Data button. (Create custom data set button should be a separate story as I don’t know if that will make it into 1.0. Featured Downloads should also be a separate story)
- [ ] #160: New issue for the “Create Custom Data Set” button.
-  [x] #157: Data Insights
-  [ ] #159: Data At A Glance – These would just be expand collapse sections. The title of the unit would be the Data Category Name (e.g. Earnings) and the information contained in the expanded container would be a list of  the individual metrics (i.e. average salary 10 years after leaving school) in plain English.
- [ ] #265: See the Data in Action
- [x] #264: Join the Conversation
- [ ] #160: Featured Downloads - Second part of #160. Let me know if I need to break this into a separate issue.
- [ ] #262: Create Custom Data Set - Per above new issues #262
- [ ] #158: Layman’s Video (deferred from 1.0)

**Comment by barkimedes** [comment]
@diego- I gave it checkboxes and checked the issues that seemed to be done. The one that I was confused about was #160. It's closed, but I couldn't tell if it was *done*. So anywho, if someone else wants to take a look and see what the real status of this epic is, that'd be swell.

**Comment by LisaGee** [comment]
I think the bulk of this Epic is done. The outstanding items all have their own stories. Closing the Epic
